tests/unit/#155.test.scala

```diff
@@ -0,0 +1,71 @@
+// SYNTAX TEST "source.scala"
+
+    transparent inline def foo
+//  ^^^^^^^^^^^ storage.modifier.other
+//              ^^^^^^ storage.modifier.other
+
+    transparent inline xdef foo
+//  ^^^^^^^^^^^ - storage.modifier.other
+//              ^^^^^^ - storage.modifier.other
+
+    transparent inline defx foo
+//  ^^^^^^^^^^^ - storage.modifier.other
+//              ^^^^^^ - storage.modifier.other
+
+    transparent inline final def assert
+//  ^^^^^^^^^^^ storage.modifier.other
+//              ^^^^^^ storage.modifier.other
+
+    transparent badkeyword inline override def alternative
+//  ^^^^^^^^^^^ storage.modifier.other
```

Comment:
Related to the previous comment: I don't think that we should have a test requiring that `transparent` is `storage.modifier.other` in this situation. The general rule of thumb that I've been applying is that when it's invalid Scala, it can be highlighted as anything; we shouldn't require specific highlightings for invalid Scala in our tests, because that could make future refactorings more difficult.

Reply:
This is just a consequence of the `[a-z\\s]*` above.

Reply:
This might be practical if we have
```
transparent inilne def alternative
```
In this case `transparent` and `def` are highlighted but not `inilne` which makes it trivial to find the typo.

Reply:
Okay, that's actually a quite convincing argument for keeping `[a-z\\s]*`. In that case, we can merge the PR.